homeworks/README.md

```diff
@@ -0,0 +1,41 @@
+# Installation instructions
+To complete the assignments, you need a working python installation with jupyter and the following libraries: numpy, scipy, matplotlib, 
+scikit-learn, pytorch, torchvision and pillow. We recommend using anaconda although this is not mandatory.
+
+## Installing anaconda
+You can Anaconda installation instructions here https://docs.anaconda.com/anaconda/install/index.html .
+
+Once anaconda is installed, create a virtual environment dedicated to the homeworks using the following command
+```
+conda create --name deep_learning
+```
+You can activate the environment using the command
+```
+source activate deep_learning
+```
+and deactivate it using
+```
+conda deactivate
+```
+## Installing pytorch

```

Comment:
N'ajouterait-on pas "Be sure to activate your environment with `source activate deep_learning` before entering the install command." ?